packages/dci-api/src/validations.ts

```diff
@@ -37,20 +37,32 @@ const authorize = z.object({
     .optional()
 })
 
-const languageCode = z.string()
+const languageCode = z.string().regex(/^[a-z]{3,3}$/)
```

Comment:
hmm.. could we get the OpenCRVS-supported locales here some way? Could we read some config and then only allow 'en/fr' for example 🤔

Reply:
well none of our data from the form has a different locale, so it doesn't make any difference at the moment I think

Reply:
Yeah, makes sense. Lets just be aware that the locale doesn't do anything, or lets remove it from request / response soon.